doc/kick/description.md

```diff
@@ -0,0 +1,3 @@
+The `kick` command will begin a poll to kick a specified user.
+
+To successfully kick a user, a two thirds majority of the users in the current voice channel is required. By invoking this command, an upvote will automatically be cast for the invoker. Other users may vote using the `vote` command.
```

Comment:
I think you meant to say that a two thirds majority of the users in the ***server*** are required.